lesson14/main.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+const wrapper = document.getElementById("wrapper");
+const lists = document.getElementById("lists");
+const createElBtn = document.getElementById("create-element-btn");
+createElBtn.classList.add("btn");
+const modal = document.getElementById("modal");
+const displayModal = document.getElementById("display-modal");
+const jsonURL = "https://myjson.dit.upm.es/api/bins/j16z";
+const removeModal = () => {
+  modal.remove();
+  wrapper.classList.remove("bg_gray");
+};
+
+const loadingIcon = document.createElement("img");
+const startLoading = () => {
+   loadingIcon.id = "loadingIcon";
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+
+const endLoading = () => loadingIcon.remove();
```

Comment:
IMO
It is best to avoid defining constants globally as much as possible. There is a possibility that an unsuitable tag will be generated.
It is also more readable to define constants where they are used.
```suggestion
const startLoading = () => {
  const loadingIcon = document.createElement("img");
  loadingIcon.id = "loadingIcon";
  loadingIcon.src = "img/loading-circle.gif";
  wrapper.appendChild(loadingIcon);
};

const endLoading = () => {
  const loadingIcon = document.getElementById("loadingIcon");
  loadingIcon.remove();
};
```

Reply:
Thank you for review.
I was confused about that point. Thank you for teaching about that.
I avoided loadingIcon constants to be globaly.
16f442b